src/Confluent.SchemaRegistry/CachedSchemaRegistryClient.cs

```diff
@@ -133,7 +134,10 @@ private static SubjectNameStrategyDelegate GetValueSubjectNameStrategy(IEnumerab
         /// <param name="authenticationHeaderValueProvider">
         ///     The authentication header value provider
         /// </param>
-        public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> config, IAuthenticationHeaderValueProvider authenticationHeaderValueProvider)
+        /// <param name="proxy">
+        ///     The proxy server to use for connections
+        /// </param>
+        public CachedSchemaRegistryClient(IEnumerable<KeyValuePair<string, string>> config, IAuthenticationHeaderValueProvider authenticationHeaderValueProvider, IWebProxy proxy = null)
```

Comment:
Please add a constructor, with the exact previous parameters, that calls this one, for binary compatibility.